app/api/import/importLocationsApi.ts

```diff
@@ -1,3 +1,4 @@
+import 'text-encoding-polyfill';
```

Comment:
Can you import this in `app.js` instead? We have a similar pollyfill there: 
```js
import 'array-flat-polyfill';
```

Reply:
Done